com.dropbox.Client.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id": "com.dropbox.Client",
+    "runtime": "org.gnome.Platform",
```

Comment:
GitHub doesn't let you comment commit messages, but the commit message needs rewording here to match the decision to use org.gnome.Platform in the end.

Reply:
Sure